src/deblock.rs

```diff
@@ -1350,25 +1359,30 @@ pub fn deblock_plane<T: Pixel>(
 
 // sse count of all edges in a single plane, accumulates into vertical and horizontal counts
 fn sse_plane<T: Pixel>(
-  rec: &Plane<T>, src: &Plane<T>, v_sse: &mut [i64; MAX_LOOP_FILTER + 2],
-  h_sse: &mut [i64; MAX_LOOP_FILTER + 2], pli: usize, blocks: &FrameBlocks,
-  bd: usize
+  fi: &FrameInvariants<T>, rec: &Plane<T>, src: &Plane<T>, v_sse: &mut [i64; MAX_LOOP_FILTER + 2],
+  h_sse: &mut [i64; MAX_LOOP_FILTER + 2], pli: usize, blocks: &FrameBlocks
 ) {
   let xdec = rec.cfg.xdec;
   let ydec = rec.cfg.ydec;
 
+  // Deblocking happens in 4x4 (luma) units; luma x,y are clipped to
+  // the *crop frame* by 4x4 block.
+  let cols = cmp::min(blocks.cols, (fi.width + MI_SIZE - 1) >> MI_SIZE_LOG2);
```

Comment:
So, we cannot just use : cols = (fi.width + MI_SIZE - 1) >> MI_SIZE_LOG2 ? Since it seems always <=  blocks.cols.
And same for rows?

Reply:
*probably*  
Something in the back of my head said 'be paranoid'.  You're probably right.

Reply:
Addressed by PR 1366